src/frontend/apps/impress/src/pages/docs/[id]/index.tsx

```diff
@@ -22,6 +23,9 @@ export function DocLayout() {
 
   return (
     <MainLayout withoutFooter>
+      <Head>
+        <meta name="robots" content="noindex" />
+      </Head>
```

Comment:
Is it possible to put it above MainLayout ? I know it is working but I find that disturbing to have it has a child of MainLayout.